Pod/Classes/PINRemoteImageManager.m

```diff
@@ -189,7 +188,7 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
         if (!configuration) {
             configuration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
         }
-        _callbackQueue = dispatch_queue_create("PINRemoteImageManagerCallbackQueue", DISPATCH_QUEUE_CONCURRENT);
+        _callbackQueue = dispatch_get_main_queue();
```

Comment:
I think this should only be set once for now through the initializer. If there is a good reason to swap the callback queue at runtime we can revisit but in the meanwhile this will satisfy the thread-safety concerns here.